tests/unit/openwrt/golden/ipq806x/root/bin/config-version.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+usage(){
+  cat << EOF
+usage: "$(basename "$0")" [OPTIONS] ARGS
+Set OpenWRT config to specific version
+OPTIONS:
+  -h      Show this message
+  -c      Configuration version [default: 0]
+EXAMPLES:
+  To set config version 1:
+      "$(basename "$0")" -c 1
+EOF
+}
+
+CONFVER=0
+FILES=(wireless network)
+CONFROOT='/etc/config'
+
+while getopts "h:c:" OPTION
+do
+  case $OPTION in
+    c )
+      CONFVER=$OPTARG
+      ;;
+    h )
+      usage
+      exit 0
+      ;;
+    \? )
+      usage
+      exit 1
+      ;;
+  esac
+done
+shift $((OPTIND-1))
+
+RELOAD=0
+
+for file in ${FILES[*]}
+do
+  # exit if the target version doesn't exist
+  if [ ! -f "$CONFROOT/$file.$CONFVER" ]
+  then
+    echo "no file $CONFROOT/$file.$CONFVER"
+    exit 1
+  fi
+  # create symlinks if the current version isn't active
+  if [ ! -L "$CONFROOT/$file" ] || [ ! "$(readlink $CONFROOT/$file)" = "$CONFROOT/$file.$CONFVER" ]
+  then
+    rm -f $CONFROOT/$file
```

Comment:
to be on the safe side, can we verify that this is a symlink before removing it?

Reply:
definitely

Reply:
Please incorporate that prior to merging this PR as well.

Reply:
@davidelang @owendelong  heres the check: https://github.com/socallinuxexpo/scale-network/pull/203/files#diff-009c980269baa7501260e6c3f1b9e298R51-R53